cpp/include/games/GameTransforms.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <core/concepts/Game.hpp>
+
+/*
+ * This file contains game transformations that can be applied to a game to modify its behavior.
```

Comment:
I would change this sentence to read,

"This file contains metafunctions that create Game types from other Game types."